src/test/java/fi/otavanopisto/kuntaapi/test/ManagementPageMocker.java

```diff
@@ -0,0 +1,91 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static org.junit.Assert.fail;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import fi.metatavu.management.client.model.Page;
+
+public class ManagementPageMocker extends AbstractBaseMocker {
+
+  private static final String PATH_TEMPLATE = "%s/%s";
+  private static final String PAGES_PATH = "/wp-json/wp/v2/pages";
+  
+  private ResourceMocker<Integer, Page> pageMocker = new ResourceMocker<>();
+
+  public ManagementPageMocker() {
+    mockList();
+  }
+  
+  @Override
+  public void startMock() {
+    super.startMock();
+    pageMocker.start();
+  }
+  
+  @Override
+  public void endMock() {
+    pageMocker.stop();
+    super.endMock();
+  }
+  
+  /**
+   * Mocks management pages
+   * 
+   * @param ids page ids
+   * @return mocker
+   */
+  public ManagementPageMocker mockPages(Integer... ids) {
+    try {
+      for (Integer id : ids) {
+        if (!pageMocker.isMocked(id)) {
+          Page page = readPageFromJSONFile(String.format("management/pages/%d.json", id));
+          mockPage(page);
+        } else {
+          pageMocker.setStatus(id, MockedResourceStatus.OK);
+        }
+      }
+    } catch (JsonProcessingException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)